nsv13/code/modules/overmap/camera.dm

```diff
@@ -60,6 +60,7 @@
 	if(eyeobj?.off_action)
 		qdel(eyeobj.off_action)
 	M.cancel_camera()
+	M.remote_control = null
```

Comment:
This likely fixes the issue but can you explain to Fran why the code qdel's above?